plugins/modules/web_infrastructure/jira.py

```diff
@@ -36,15 +36,22 @@
 
   username:
     type: str
-    required: true
+    required: false
     description:
-      - The username to log-in with.
+      - The username to log-in with. Must be used with password.  Mutually exclusive with token.
```

Comment:
Let's make this two paragraphs:
```suggestion
      - The username to log-in with.
      - Must be used with I(password). Mutually exclusive with I(token).
```